src/components/Skeleton/Skeleton.module.css

```diff
@@ -30,7 +30,7 @@
 }
 
 .skeleton.skeleton-circle {
-  border-radius: var(--eds-border-radius-round);
+  border-radius: var(--eds-border-radius-full);
```

Comment:
`-round` only works when the object is square. Else it generates an oblique spheroid! `-full` always does what we intended round to do... form a pill-shaped rounded corner